test/no-duplicate-records.ts

```diff
@@ -19,41 +19,29 @@ describe('Migrations Collection', () => {
     await coll.deleteMany({});
   });
 
-  it('should run migrations and only record them once', (done: Mocha.Done) => {
+  it('should run migrations and only record them once', async () => {
```

Comment:
This test saved me from a terrible bug where I removed literally the key functionality of the library (keeping track of which migrations have been run, and not running them again) while fixing another test.